config/config.go

```diff
@@ -34,12 +34,12 @@ type Field struct {
 }
 
 // NewField constructor.
-func NewField(fld *reflect.StructField, val *reflect.Value) (*Field, error) {
+func NewField(prefix string, fld *reflect.StructField, val *reflect.Value) (*Field, error) {
```

Comment:
@mantzas could you please clarify a little bit? Do we need to have this constructor exported? I haven't found any use-cases across this repository, but since harvester is being used by others, it could be risky to change the parameters list here. If we can't make it hidden - perhaps I'll introduce different constructor with prefix support.

Reply:
@Oberonus You are correct, nobody should use it outside harvester so if it does not break code, pls proceed.

Reply:
cool, thank you for clarification